lib/claide/argv.rb

```diff
@@ -209,14 +209,16 @@ def all_options(name)
       options
     end
 
-    private
+    protected
 
     # @return [Array<Array<Symbol, String, Array>>] A list of tuples for each
     #         non consumed parameter, where the first entry is the `type` and
     #         the second entry the actual parsed parameter.
     #
     attr_reader :entries
```

Comment:
private causes a warning, it can be either public or protected to resolve the warning. Is public preferable?